pkg/devfile/adapters/docker/utils/utils.go

```diff
@@ -209,33 +208,25 @@ func containerHasPort(devfilePort nat.Port, exposedPorts nat.PortSet) bool {
 func UpdateComponentWithSupervisord(comp *common.DevfileComponent, runCommand common.DevfileCommand, supervisordVolumeName string, hostConfig *container.HostConfig) {
 
 	// Mount the supervisord volume for the run command container
-	for _, action := range runCommand.Actions {
-		if *action.Component == *comp.Alias {
-			AddVolumeToContainer(supervisordVolumeName, adaptersCommon.SupervisordMountPath, hostConfig)
-
-			if len(comp.Command) == 0 && len(comp.Args) == 0 {
```

Comment:
Wouldn't it impact on functionality? I am aware that v2 does not have `command` `args`

Reply:
I am not sure about this. We need to ask someone who has worked on this file previously.

Reply:
@maysunfaisal @johnmcollier Can you please validate this?

Reply:
@mik-dass for now can you comment this code, instead of removing and mark as `TODO`, i need these changes.

Reply:
Commented out the code